frontend/src/browser/users/ActivityPointsPage.js

```diff
@@ -0,0 +1,351 @@
+                <div className="box-tools">
+                  <div className="input-group input-group-sm" style={{ width: '150px' }}>
+                    <input
+                      type="text"
+                      name="table_search"
+                      className="form-control pull-right"
+                      placeholder="Search"
+                      {...fields.usersFilter}
+                    />
+                    <div className="input-group-btn">
+                      <button type="submit" className="btn btn-default">
+                        <i className="fa fa-search"></i>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div className="box-body table-responsive no-padding">
+                <div className="col-md-12" style={{ margin: "22px 0" }}>
+                  <div><strong><FormattedMessage {...messages.otherStudents} />:</strong></div>
+                  <div style={{ maxHeight: '300px', overflowY:'auto' }}>
+                    {!users ?
+                      <div></div>
+                    :
+                      <table className="table table-bordered">
+                        <tbody>
+                          <tr>
+                            <th><FormattedMessage {...messages.order} /></th>
+                            <th><FormattedMessage {...messages.name} /></th>
+                            <th><FormattedMessage {...messages.level} /></th>
+                            <th><FormattedMessage {...messages.gainedPoints} /></th>
+                            <th><FormattedMessage {...messages.percentageGainedPoints} /></th>
+                          </tr>
+                          {filteredUsers.sort((a,b) => (a.gainedActivityPoints / a.potentialActivityPoints > b.gainedActivityPoints / b.potentialActivityPoints) ? 0 : 1).toList().map((user, i) => {
+                            if(studentLevels.has(user.studentLevelId)) {
+                              return (
+                                <tr key={user.id}>
+                                  <td>{i+1}</td>
+                                  <td>{`${user.firstName} ${user.lastName}`}</td>
+                                  <td>{studentLevels.has(user.studentLevelId) 
+                                    ? studentLevels.get(user.studentLevelId).name
+                                    : '?'}</td>
+                                  <td>{user.gainedActivityPoints}</td>
+                                  <td>{Math.round(user.gainedActivityPoints / user.potentialActivityPoints * 100)} %</td>
+                                </tr>
+                              );
+                            }
+                          })}
+                        </tbody>
+                      </table>
+                    }
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
Cele mi to pride ako prilis velka komponenta, rozbil by som to na mensie casti, minimalne na tie dve tabulky